packages/kit/src/core/build/index.js

```diff
@@ -491,15 +431,42 @@ async function build_server(
 
 	print_config_conflicts(conflicts, 'kit.vite.', 'build_server');
 
-	await vite.build(merged_config);
+	const output = /** @type {import('rollup').OutputChunk[]} */ (
+		/** @type {any} */ (await vite.build(merged_config)).output
+	);
+
+	/** @type {Record<string, string[]>} */
```

Comment:
maybe this and the next ~20 lines could be refactored out into something like a `get_methods` function?